src/extensions.web/Microsoft.Identity.Client.Extensions.Web/Microsoft.Identity.Client.Extensions.Web.csproj

```diff
@@ -0,0 +1,94 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworkNetDesktop>net461</TargetFrameworkNetDesktop>
+    <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
+    <TargetFrameworkNetCore2>netcoreapp2.2</TargetFrameworkNetCore2>
+    <TargetFrameworkNetCore3>netcoreapp3.0</TargetFrameworkNetCore3>
+
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+      $(TargetFrameworkNetDesktop);$(TargetFrameworkNetStandard);$(TargetFrameworkNetCore2)
+    </TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Linux')) Or $([MSBuild]::IsOsPlatform('OSX')) Or '$(NetCoreOnly)' !='' ">
+      $(TargetFrameworkNetStandard);$(TargetFrameworkNetCore2)
+    </TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- The MSAL.snk has both private and public keys -->
+    <DelaySign>false</DelaySign>
+  </PropertyGroup>
+
+  <PropertyGroup Label="NuGet and AssemblyInfo metadata">
```

Comment:
I assume that `msbuild /t:pack` will simply build 2 nuget packages - one for msal, one for the extensions. There will be a bit of work on the build around this, let me know if I can help.

Reply:
right clicking on extensions.web and saying "pack" definitely builds the nuget package, i've verified that.  build will definitely need some work to add the additional package.  i'm going to do the build work later once there's something to build and since i'm also going to add in the msal storage extensions here (for public client) i'll do the build work all in one sweep with all of the packages.